### PR TITLE
Add Toshi Dev target and profile.

### PR DESCRIPTION
**Pull request by roberthein**
Add a new **target**:
screen shot 2017-08-23 at 16 39 59

Add new **app icons**:
screen shot 2017-08-23 at 15 44 48

Implement different **tint-color** for `Toshi Dev`:
img_3752

### CODE REVIEW

**Comment by heumn** [comment]
I am stealing that tint color idea! Neat!

File: Toshi/Theme/Theme.swift [hunk 1]
```diff
@@ -57,7 +57,11 @@ extension Theme {
     }
 
     public static var tintColor: UIColor {
-        return UIColor(hex: "01C236")
+        #if TOSHIDEV
```
Comment (yuliaveres) [inline Toshi/Theme/Theme.swift]: great ❤️

File: Toshi/Resources/DistributionDev.plist [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+				<dict/>
+				<key>KitName</key>
+				<string>Crashlytics</string>
+			</dict>
+		</array>
+	</dict>
+	<key>ITSAppUsesNonExemptEncryption</key>
+	<false/>
+	<key>InitialGreetingAddress</key>
+	<string>0xdc1eb58ae581c2e70dd5af5c454851fa2b24acd7</string>
```
Comment (yuliaveres) [inline Toshi/Resources/DistributionDev.plist]: correct 👍

**Comment by yuliaveres** [comment]
i think the purpose of separate app is to enable particular users to switch networks;
so please, include `TOSHIDEV ` into switching logic of `NetworkSwitcher` and `SettingsController`.

**Comment by yuliaveres** [comment]
the rest is ok, TOSHIDEV is on Crashlytics as well 👍

**Comment by roberthein** [comment]
Can you check if everything is correct @faymontage?

**Pull request merged**